tests/field/test_magneticfields.py

```diff
@@ -474,6 +474,52 @@ def test_Dommaschk(self):
         field_json_str = json.dumps(SIMSON(Bfield), cls=GSONEncoder)
         Bfield_regen = json.loads(field_json_str, cls=GSONDecoder)
         self.assertTrue(np.allclose(B, Bfield_regen.B()))
+        
+        #Field configuration from Dommaschk paper equation number (40)
+        mn = [[5, 2], [5, 4], [5, 10]]
+        coeffs = [[1.4, 1.4], [19.25, 0], [5.10e10, 5.10e10]]
+        Bfield = Dommaschk(mn=mn, coeffs=coeffs)
+        point = np.asarray([[0.71879008, 0.76265643, 0.0745]])
+        Bfield.set_points(point)
+        gradB = np.array(Bfield.dB_by_dX())
+        transpGradB = np.array([dBdx.T for dBdx in gradB])
+        B = Bfield.B()
+        print(point)
+        print(B)
+        print(gradB)        
```

Comment:
Could you just remove these prints? I can then approve it and merge it if everyone is ok with it